Doc/library/configparser.rst

```diff
@@ -136,6 +136,26 @@ sections [1]_.  Note also that keys in sections are
 case-insensitive and stored in lowercase [1]_.
 
 
+If we need to read several configurations, each one having more priority than the previous one, we can use the same :class:`ConfigParser` instance to override previous defined data and keep not redefined data.
```

Comment:
Am not a native English speaker but this "keep not redefined data" phrase is confusing at least to me. Not sure how you can improve it.

Reply:
Thanks for the contribution @sblondon!

A simple grammatical improvement would be to change "keep not redefined data" to "not keep redefined data". However, I would suggest a few other changes to this part as well. This retains the same meaning, but significantly improves the phrasing:
```suggestion
   If it is necessary to read several configurations, with each one having more 
   priority than the previous one, an instance of :class:`ConfigParser` can be 
   used to override previously defined properties and retain existing ones.
```
reST and Sphinx specifications (this applies to the other sections as well):
1) Each line needs to start with 3 space characters.
2) The maximum number of characters per line is 80 characters. 

Significant phrasing changes:
1) In technical writing, it is preferable to avoid using second person pronouns, such as "we" when possible.
2) "data" -> "properties". This term is more of an accurate descriptor than "data". An alternative term would be "attributes".
3) "keep not redefined data" -> "retain existing ones". This removes redundancy, "ones" refers to the previously mentioned properties, which implies "retain existing properties".